src/index.ts

```diff
@@ -34,7 +43,16 @@ const manager: JupyterFrontEndPlugin<ISecretsManager> = {
     connector: ISecretsConnector
   ): ISecretsManager => {
     console.log('JupyterLab extension jupyter-secrets-manager is activated!');
-    return new SecretsManager({ connector });
+    const secretsManager = new SecretsManager({ connector });
+    const panel = new Panel();
```

Comment:
We should probably build the side panel in another plugin, to be able to disable it.

Reply:
Is it too much to have the UI in another plugin? Do you have a use case in mind where we want to enable the secrets manage functionality but disable the UI?

Reply:
It already works to automatically populate inputs without UI, see https://github.com/jupyterlab-contrib/jupyter-secrets-manager/pull/8#issuecomment-2767228963.